apps/next/pages/providers.tsx

```diff
@@ -1,17 +1,19 @@
 import { Seo } from '@/components/Seo'
-import { ManagerProvider } from '@public-assembly/dao-utils'
+import { ManagerProvider, GovernorProvider } from '@public-assembly/dao-utils'
 import { isServerSide } from 'utils/helpers'
 import { PrintManagerProviderData } from '../components'
 
 function Providers() {
-  const tokenAddress = '0x8983ec4b57dbebe8944af8d4f9d3adbafea5b9f1' as `0x${string}`
+  const tokenAddress = '0xdf9b7d26c8fc806b1ae6273684556761ff02d422' as `0x${string}`
```

Comment:
Any string constants are best declared as all caps and snake case preferably in their own file for constants to keep track of them or at least contextually in the same file at the top.